packages/service-clients/azure-client/src/AzureClient.ts

```diff
@@ -59,6 +59,29 @@ const azureClientFeatureGates = {
 	"Fluid.Container.ForceWriteConnection": true,
 };
 
```

Comment:
@scottn12 - I think I need to update OdspClient too, for parity (even though there'd be no v1/v2 compat issues). Anywhere else?  To cover all Declarative model scenarios.

Reply:
I should hopefully have an answer for you Tuesday afternoon after syncing with a PM. Let me know if you would like to be added onto the meeting!

Reply:
No thanks just lmk :)

Reply:
@scottn12  any update?

Reply:
Nothing concrete yet, @markfields. I did however raise your concerns with @sashasimic and Pranshu, so hopefully we should be able to get some guidance on this soon.

Reply:
Hey @markfields, quick update. We will want to duplicate these configs in OdspClient, but I will open a separate ticket for that, so you don't need to worry about it in this PR.

Reply:
I'm doing it here to get it over with.